src/utils/getChildPageNodes.ts

```diff
@@ -7,7 +7,7 @@ import { PageNode } from 'src/directory/directory';
  * @returns
  */
 export const getChildPageNodes = (route: string) => {
-  return traverseDirectory(route, directory as PageNode);
+  return traverseDirectory(route, directory as PageNode) || [];
```

Comment:
A fallback value is needed here if we are ever going to have static files that don't have entries in the directory.